vaadin-testbench-unit/src/main/java/com/vaadin/flow/component/radiobutton/RadioButtonGroupWrap.java

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright (C) 2022 Vaadin Ltd
+ *
+ * This program is available under Commercial Vaadin Developer License
+ * 4.0 (CVDLv4).
+ *
+ *
+ * For the full License, see <https://vaadin.com/license/cvdl-4.0>.
+ */
+package com.vaadin.flow.component.radiobutton;
+
+import java.util.Map;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import org.jetbrains.annotations.NotNull;
+
+import com.vaadin.flow.component.ItemLabelGenerator;
+import com.vaadin.flow.data.binder.HasItemComponents;
+import com.vaadin.testbench.unit.ComponentWrap;
+import com.vaadin.testbench.unit.Wraps;
+
+/**
+ * Test wrapper for RadioButtonGroup components.
+ *
+ * @param <T>
+ *            component type
+ * @param <V>
+ *            value type
+ */
+@Wraps(fqn = "com.vaadin.flow.component.radiobutton.RadioButtonGroup")
+public class RadioButtonGroupWrap<T extends RadioButtonGroup<V>, V>
+        extends ComponentWrap<T> {
+    /**
+     * Wrap given component for testing.
+     *
+     * @param component
+     *            target component
+     */
+    public RadioButtonGroupWrap(T component) {
+        super(component);
+    }
+
+    @Override
+    public boolean isUsable() {
+        return super.isUsable() && !getComponent().isReadOnly();
+    }
+
+    /**
+     * Selects an item by its client string representation.
+     *
+     * @param selection
+     *            item string representation
+     */
+    public void selectItem(String selection) {
+        ensureComponentIsUsable();
+        updateSelection(selection);
+    }
+
+    /**
+     * Deselects item selection.
+     */
+    public void deselectItem() {
+        ensureComponentIsUsable();
+        deselectAll();
+    }
+
+    /**
+     * Deselects all client usable items.
+     */
+    public void deselectAll() {
```

Comment:
I would remove the `deselectAll` method, since radio button group can have a single selection

Reply:
gone